### DISCUSSION TITLE
disable drag & resize region

### DISCUSSION THREAD

**Discussion by Smitt2767** [katspaugh/wavesurfer.js#3250] [General]
I have a situation where I want to disable the drag & resize of a particular region. I have achieved it using the region setOptions method but by doing this it only removes those listeners but div with a data-resize attribute is not removed from dom. Is there a better way to do it instead of the setOptions method?

Ex:- https://codesandbox.io/s/reactjs-playground-forked-gk9ywx?file=/src/index.js

**Comment by katspaugh** [comment]
There's an open issue for that: https://github.com/katspaugh/wavesurfer.js/issues/3235
